examples/getting-started-movielens/03-Training-with-PyTorch.ipynb

```diff
@@ -37,52 +37,44 @@
     "\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #6.        train_loss, y_pred, y = process_epoch(train_loader,
</pre><p>I recommend you set the loss as BCE because the process_epoch's default loss func is MSE loss which does not make sense for a binary classification problem: https://github.com/NVIDIA-Merlin/NVTabular/blob/main/nvtabular/framework_utils/torch/utils.py#L64</p><p> </p><p>so training a model with wrong loss wont be accurate. what do you think?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/Merlin/pull/779/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

Reply:
<p>that's a good point, change made</p>